napari/layers/utils/layer_utils.py

```diff
@@ -233,3 +234,101 @@ def compute_multiscale_level(
     else:
         level = 0
     return level
+
+
+def stack_to_images(stack, axis):
+    """Function to split the active layer into separate layers along an axis
+
+    Parameters
+    ----------
+    stack : napari.layers.Image
+        The image stack to be split into a list of image layers
+    axis : int
+        The axis to split along.
+
+    Returns
+    -------
+    list
+        List of images
+    """
+
+    # if not isinstance(stack, Image):
+    #     print("Active layer is not an image")
+    #     return None
+
+    data = stack.data
+    name = stack.name
+
+    cmaps = list(simple_colormaps.keys())
+    num_dim = len(data.shape)
+
+    if num_dim < 3:
+        print("not enough dimensions")
+        return None
+
+    if axis >= num_dim:
+        print("the image has {} dimensions".format(num_dim))
+        return None
+
+    imagelist = list()
+
+    ImageLayer = type(stack)
+    for i in range(data.shape[axis]):
+        layer_name = "{:02d}_{}".format(i, name)
+
+        try:
+            color = cmaps[i]
+        except IndexError:
+            color = 'gray'
+
+        image = ImageLayer(
+            np.take(data, i, axis=axis),
+            blending='additive',
+            colormap=color,
+            name=layer_name,
+        )
+
+        imagelist.append(image)
+
+    return imagelist
+
+
+def images_to_stack(images, axis=-1):
+    """Function to combine selected image layers in one layer
+
+    Parameters
+    ----------
+    viewer : napari.viewer.Viewer
+        The viewer with the selected image
+    axis : int
+        Index to to insert the new axis
+
+    Returns
+    -------
+    stack : napari.layers.Image
+        Combined image stack
+    """
+
+    nc = len(images)
+    if nc < 2:
+        print("less than two layers selected")
+        return None
+
+    new_list = list()
+    for i in range(nc):
+        # if not isinstance(images[i], Image):
+        #     print("Selected layer {} is not an image".format(images[i].name))
+        #     return None
+        if i == 0:
+            shape = images[i].data.shape
+        else:
+            if shape != images[i].data.shape:
+                print("Layers not the same size")
+                return None
+        new_list.append(images[i].data)
+
+    new_data = np.stack(new_list, axis=axis)
+
+    stack = type(images[0])(new_data)
```

Comment:
Oh I see what's happened here, this must be the spot where circular imports were making things tricky. @sofroniewn should we shift all of this someplace else?

Reply:
Would it help if things were in our https://github.com/napari/napari/blob/master/napari/layers/image/_image_utils.py?

Reply:
I tried using _image_utils.py, but it looks like _image_utils.py is imported when Image is imported:
```~/Code/napari/napari/layers/image/__init__.py in <module>
----> 1 from .image import Image

~/Code/napari/napari/layers/image/image.py in <module>
     11 from ..intensity_mixin import IntensityVisualizationMixin
     12 from ._image_constants import Interpolation, Interpolation3D, Rendering
---> 13 from ._image_utils import guess_rgb, guess_multiscale
     14
     15

~/Code/napari/napari/layers/image/_image_utils.py in <module>
      3
      4 import numpy as np
----> 5 from napari.layers.image.image import Image
      6 from napari.layers.labels.labels import Labels
      7 from vispy.color import Colormap
```